ci-operator/config/openshift/jenkins-openshift-login-plugin/openshift-jenkins-openshift-login-plugin-master.yaml

```diff
@@ -38,10 +38,12 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: e2e-aws-jenkins
+- as: e2e-aws-jenkins-sync-plugin
```

Comment:
the sync plugin e2e always dumps jenkins job logs in its runs

for example, https://github.com/openshift/jenkins-sync-plugin/blob/a1665ff237fbc2fd9982669b3ab7519998d16f30/test/e2e/sync_plugin_test.go#L1001

this retrieval of job logs employs use of the service account token to authenticate with jenkins, which in turn leverages the login plugin and its integration with openshift oauth

i.e. https://github.com/openshift/jenkins-sync-plugin/blob/73e7999555eb5b107aa528ef791146095e1d5376/test/e2e/ref.go and any use of 

`cmd := fmt.Sprintf("TOKEN=`cat /var/run/secrets/kubernetes.io/serviceaccount/token` && curl -X GET -H \"Authorization: Bearer $TOKEN\" %s", fullURL)`

this use of getting job logs is what we got from the old e2e-aws-jenkins for validating any login plugin PRs